File: backend/spring/ain/src/main/java/com/ssafy/ain/global/constants/SuccessCode.java
package com.ssafy.ain.global.constants;

import lombok.AllArgsConstructor;
import lombok.Getter;
import org.springframework.http.HttpStatus;

import static org.springframework.http.HttpStatus.*;

@AllArgsConstructor
@Getter
public enum SuccessCode {

    //회원
    GET_MEMBER(OK, "회원 정보를 조회하는데 성공했습니다!");

    private final HttpStatus status;
    private final String message;
}

File: backend/fastapi/ain/total/constant/SuccessCode.py
from enum import Enum


class SuccessCode(Enum):
    # 챗봇
    CREATE_IDEAL_PERSON_CHATBOT = "이상형 챗봇을 생성하는데 성공했습니다!"
    DELETE_IDEAL_PERSON_CHATBOT = "이상형 챗봇을 삭제하는데 성공했습니다!"

    # 채팅
    CREATE_IDEAL_PERSON_CHAT = "이상형에게 채팅 메시지를 전송하는데 성공했습니다!"